csrankings-z.csv

```diff
@@ -293,7 +293,7 @@ Zilin Jiang,Arizona State University,https://www.zilin.one,CU1rungAAAAJ
 Zilong Liu 0001,University of Essex,https://www.essex.ac.uk/people/liuzi11203/zilong-liu,ODFLmIwAAAAJ
 Zimeng Zhou,Shandong University,https://faculty.sdu.edu.cn/zhouzimeng1/en/index.htm,r5EK8wMAAAAJ
 Ziming Zhao 0001,University at Buffalo,http://zzm7000.github.io,l6M4h_YAAAAJ
-Zimu Zhou,Singapore Management University,https://zhouzimu.github.io,RUF3lHQAAAAJ
+Zimu Zhou,City University of Hong Kong,https://zhouzimu.github.io,RUF3lHQAAAAJ

```

Comment:
Please clarify how you meet the eligibility criteria (you must be able to solely supervise a CS student for a PhD _in Computer Science_).